src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java

```diff
@@ -123,15 +128,64 @@ private void initializeButtons() {
                 smallStatusLabel.setStyle("");
                 bigStatusLabel.setStyle("-fx-text-fill: red;");
             }
+            
+            switchLive();
+            
+            updateTables(live);
         });
 
         btnBack.toFront();
         btnBack.setOnAction((event) -> {
             MenuController.show();
         });
+        
+        btnNext.setOnAction((event) -> {
+            script.next();
+            updateTables(live);
+        });
     }
 
     /**
+     * Updates the table contents according to the current position in the script.
```

Comment:
Noticed we didn't need to pass around the live parameter, so I removed it entirely.